### PR TITLE
[Nuxt 3] migrate /metal page

### LINKED ISSUE #3013: Vue 3 update: Migrate metal page

### PR DESCRIPTION
**Pull request by techtolentino**
## Changes

Closes #3013 

## Implementation details
- updated component tags to use new convention 
- updated small style in `body` selector, to allow videos to `position: sticky` while scrolling
- ~added nuxt3-lazy-load package to keep lazy loading for assets~ (removed package)
- moved videos to `public` directory because they weren't working from `static`

## Screenshots

https://user-images.githubusercontent.com/6276074/224142367-eecfa958-8795-49df-88c9-b3af74acd9da.mov

### CODE REVIEW

File: components/metal/CapabilitiesSection.vue [hunk 1]
```diff
@@ -49,35 +51,32 @@
...
+  return extension === ".mp4";
+};
 </script>
 
 <style lang="scss" scoped>
 @use "~/assets/scss/carbon.scss";
-@import "~carbon-components/scss/globals/scss/typography";
+@use "~/assets/scss/helpers/index.scss" as qiskit;
+
+// @import "~carbon-components/scss/globals/scss/typography";
```
Comment (eddybrando) [inline components/metal/CapabilitiesSection.vue:79]: I believe we can remove this.

Comment (techtolentino) [reply components/metal/CapabilitiesSection.vue]: Resolved https://github.com/Qiskit/qiskit.org/pull/3019/commits/b02185b3cea9147da743470ae4c3688d0657e26c

File: package-lock.json [hunk 2]
```diff
@@ -5,6 +5,9 @@
   "packages": {
     "": {
       "hasInstallScript": true,
+      "dependencies": {
```
Comment (eddybrando) [inline package-lock.json:8]: I believe that the changes to the package-lock.json can be reverted.

Comment (techtolentino) [reply package-lock.json]: ah thanks for catching this

reverted w/ https://github.com/Qiskit/qiskit.org/pull/3019/commits/8a837d362d70de7a1944495fd27ae3c1f2ea715f

**Pull request merged**